types/src/v0/impls/state.rs

```diff
@@ -663,6 +695,41 @@ impl HotShotState<SeqTypes> for ValidatedState {
             .resolve()
             .expect("Chain Config not found in validated state");
 
+        // Validate l1_finalized.
+        let Some(proposed_finalized) = proposed_header.l1_finalized() else {
+            tracing::error!("L1 finalized has None value.");
```

Comment:
Does this mean the proposer proposed an invalid block? I think in that case this should be `warn` at most? We should only log `error` if this is something the application cannot handle and/or it's worth waking people up for.

Reply:
appears to have been addressed here: https://github.com/EspressoSystems/espresso-sequencer/pull/2089/commits/8f0d2291f21808ef96e3671a99b069b9ffcfaa31